### ISSUE TITLE
key/button binding

### ISSUE THREAD

**Issue by ceddec1001** [Tylemagne/Gopher360#69]
is it possible to bind any button on the controller  to a key on the board? :P

**Comment by Tylemagne** [comment]
Yep, you just have to open config.ini. There's a URL for keycodes, as well as a list of buttons the controller has. You just have to type it out and write in the desired keycode it's wired to.

**Comment by SyntaxB** [comment]
Can we bind 2 controller buttons to 1 keyboard button?
